.github/workflows/create_release.yml

```diff
@@ -0,0 +1,190 @@
+# Copyright (c) ONNX Project Contributors
```

Comment:
Many thanks for the comments. Exactly the use case list is still missing, or we have to work it out. 
* When do we want to start the workflow in “create_release.yml”?
* And when do we possibly only want to start the individual processes? 
with release in the name

1. That should be already implemented in my new "workflow" => In my opinion we could show also make it possible to trigger that event manuelly. For this use case, for me it would be ok, if we publish the successional wheels builds (although if not all os were successful)

2. For RC candidates I would use 
  a) a manuel event: some "configurable workflow dispatch" where we can set the branch and version
  b) or it's like a "draft a new release" from the github ui and set the version manuelly
=> Think that should be discussed

3. Official release
  a) Of course not automated. 

I would integrat "rc" into the version number, but of course that has to be discussed.